libfswatch/src/libfswatch/c++/event.hpp

```diff
@@ -31,6 +31,7 @@
 #  include <vector>
 #  include <iostream>
 #  include "../c/cevent.h"
+#  include "cxxfswatch_export.h"
```

Comment:
As in the C case, this file is not generated by the Autotools thus breaking the build